src/QA.Contribution.Test.Journey/Page/ContactUs.cs

```diff
@@ -49,52 +70,49 @@ public void ClearEmailAddress()
             emailField.Clear();
         }
 
-        public string EnterMessage()
+        public void ClearName()
         {
-            var message = Faker.Lorem.Paragraph();
-            var messageField = Driver.GetClickableElement(By.XPath(_messageLocator));
-            messageField.Clear();
-            messageField.SendKeys(message);
-            return message;
+            var nameField = Driver.GetClickableElement(By.XPath(_nameLocator));
+            nameField.Clear();
         }
 
-        public string GetErrorMessage()
+        public void EnterName(string name)
         {
-            var errorAlert = Driver.GetClickableElement(By.XPath(_errorLocator));
-            return errorAlert.Text;
+            EnterText(_nameLocator, name);
         }
 
-        public string GetSuccessMessage()
+        public void EnterPhone(string phoneNumber)
         {
-            var successAlert = Driver.GetClickableElement(By.XPath(_successLocator));
-            return successAlert.Text;
+            EnterText(_phoneLocator, phoneNumber);
         }
 
-        public string EnterPhone()
+        public void EnterSubject(string subject)
         {
-            var phoneNumber = Faker.Phone.Number();
-            var phoneField = Driver.GetClickableElement(By.XPath(_phoneLocator));
-            phoneField.Clear();
-            phoneField.SendKeys(phoneNumber);
-            return phoneNumber;
```

Comment:
all methods were following same set of steps so extracted it in a generic method `EnterText`